### DISCUSSION TITLE
Editor problem - iframe configuration

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2239] [Support] [answered]
A couple, possibly related, issues regarding media embeds. I've created separate threads but will link to both in comments.

Something's wrong with iframe embeds. After embedding an iframe, I can't get to the media tools to configure the iframe. Clicking on the iframe itself or on the media tools when focused on the iframe just throws a blank screen. A workaround is to insert an image using the media tools, then I'm able to access the media tools for the iframe.

**Comment by kyledenlinger** [comment]
Possibly related: [Full-width image problems in editor](https://github.com/pubpub/pubpub/discussions/2240)

**Comment by gabestein** [comment, marked as answer]
Thanks Kyle. I've created a [bug report](https://github.com/pubpub/pubpub/issues/2241) for the issue.

When stuff like this comes up that's an obvious bug in the future, feel free to add a bug report for it directly [via GitHub Issues](https://github.com/pubpub/pubpub/issues/new/choose). It includes a bit of a template that helps us collect needed info.
